src/puppetlabs/trapperkeeper/internal.clj

```diff
@@ -135,16 +136,14 @@
       (handle-prismatic-exception! e))))
 
 
-(defn parse-cli-args!
+(schema/defn parse-cli-args! :- common/CLIData
   "Parses the command-line arguments using `puppetlabs.kitchensink.core/cli!`.
   Hard-codes the command-line arguments expected by trapperkeeper to be:
       --debug
       --bootstrap-config <bootstrap file>
       --config <.ini file or directory>
       --plugins <plugins directory>"
-  [cli-args]
-  {:pre  [(sequential? cli-args)]
-   :post [(map? %)]}
+  [cli-args :- [(schema/maybe schema/Str)]]
```

Comment:
So this one is a little weird. This line: https://github.com/puppetlabs/trapperkeeper/blob/master/test/puppetlabs/trapperkeeper/bootstrap_test.clj#L93

Passes nil into `parse-cli-args!`, to test that a "Missing required argument for --bootstrap-config" error is raised. I couldn't find a way to pass, say, an empty string in or something like that get the same behavior. Honestly I'm not sure if that test is worth having though, it seems to only be testing the `puppetlabs.kitchensink/cli!` function.

I can't think of any legitimate reason to be passing nils into `parse-cli-args!`